cartographer/mapping_3d/scan_matching/fast_correlative_scan_matcher.h

```diff
@@ -68,7 +68,7 @@ struct Candidate {
   bool operator>(const Candidate& other) const { return score > other.score; }
 };
 
-using MatchingFunction = std::function<bool(const transform::Rigid3f&)>;
+using MatchingFunction = std::function<float(const transform::Rigid3f&)>;
```

Comment:
Is the return value documented anywhere else? Otherwise add comment to type what this float is.